### PR TITLE
HOLA MEMO PLEASE APROVAR

### PR DESCRIPTION
**Pull request by adalrada**
PLEASE

### CODE REVIEW

File: Merakis/Program.cs [hunk 1]
```diff
@@ -23,5 +23,6 @@ public static void Main(string[] args)
                     webBuilder.UseStartup<Startup>();
                 });
         //this is a test
+        //This is other test for a new brach
```
Comment (Astrack361) [inline Merakis/Program.cs:26]: Don't delete this line

**Review by Astrack361 — CHANGES_REQUESTED** [review]

File: Merakis/Program.cs (hunk 1 above)
Comment (adalrada) [reply Merakis/Program.cs:26]: DE ACUERDO

**Comment by adalrada** [comment]
APROVAR

File: Merakis/Program.cs (hunk 1 above)
Comment (Astrack361) [reply Merakis/Program.cs:26]: Don't speak spanish

Comment (adalrada) [reply Merakis/Program.cs:26]: UNDERSTOOD

Comment (Astrack361) [reply Merakis/Program.cs:26]: Cool

**Pull request merged**